extension/popup.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE>
 <html style="width: 400px; height: 400px;">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
```

Comment:
`<meta>` elements should always go inside the `<head>` element. Otherwise, this is invalid HTML. Note that modern browsers automatically fix this.